### PR TITLE
Write README

### PR DESCRIPTION
**Pull request by andy-faunadb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,100 @@
...
+    async function testPing() {
+      console.log(await rootClient.ping())
+    }
+
+    async function createDBAndGetClient() {
+      const dbName = 'prydain'
+      const dbRef = new Ref('databases', 'prydain')
+
+      try {
+        await rootClient.delete(dbRef)
```
Comment (eaceaser) [inline README.md]: I don't think the example should use async/await. Most users will not have access to these constructs.

Comment (andy-faunadb) [inline README.md]: I have an ES5 version, use that alone then? (no 'import', 'async', arrow functions)

Comment (eaceaser) [inline README.md]: I think an ES5 version there would be good. Having both couldn't hurt I guess.

**Comment by eaceaser** [comment]
other than having an es5 example everything looks good

**Pull request merged**